hashicorp/boundary-ui
# chore: 🤖 alias target route

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12959)

## Description

This PR adds a subroute within targets for the users to be able to create an alias within a target

## Screenshots (if appropriate):


 

https://github.com/hashicorp/boundary-ui/assets/15043878/a1ee4787-8ab2-441b-bcef-006bfcc362de



mirage: https://boundary-omzeos2yz-hashicorp.vercel.app/

## How to Test
case 1: 
1. Get the latest main from the boundary repo and run the controller
2. create a TCP target with an alias
3. After successful creation, it should be added to the sidebar when you click on the created target
4. clicking on the alias (sidebar) should take you to the actual alias form so the user can edit if they like

case 2: 
1. if you didn't add an alias while creating a target, the sidebar should show an option to create an alias within that target. 

 




:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
